src/elements/skiplink-list/skiplink-list.visual.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { html, type TemplateResult } from 'lit';
+
+import { describeViewports, visualDiffDefault, visualDiffFocus } from '../core/testing/private.js';
+
+import './skiplink-list.js';
+import '../link/block-link.js';
+
+const links = (): TemplateResult => html`
+  <sbb-block-link href="#">Link 1</sbb-block-link>
+  <sbb-block-link href="#">Link 2</sbb-block-link>
+  <sbb-block-link href="#">Link 3</sbb-block-link>
+`;
+
+describe(`sbb-skiplink-list`, () => {
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    it(
+      'not focused',
```

Comment:
this looks like the 'no title' but unfocused; could we have a for cycle with default/focus?